.gitignore

```diff
@@ -26,3 +26,4 @@
 /src/**/yarn-error.log
 /tmp
 /w
+*~
```

Comment:
Could you be a bit more specific here ? Add a folder ? That seems a bit broad

Reply:
`emacs` creates a lot of files which end in `~` so that it is possible to recover files. I will add a comment.